moto/cloudformation/models.py

```diff
@@ -677,6 +677,8 @@ def update_stack(self, name, template, role_arn=None, parameters=None, tags=None
 
     def list_stack_resources(self, stack_name_or_id):
         stack = self.get_stack(stack_name_or_id)
+        if stack is None:
+            return []
```

Comment:
If the stack does exist, but is simply empty, it would also return an empty list I assume?
So it might be better to simply return None here, and change _responses.py_ to explicitly check _if resources is None:_

Reply:
wfm

Reply:
That is probably more realistic; I doubt that you can create a stack with no resources.

Reply:
I think I've got something even better: raise the `ValidationError` is the stack is `None`.

Reply:
These ideas I had did not work. I will revert.